lib/bundle-validator.js

```diff
@@ -248,37 +243,6 @@ function isValidImageType(imagePath) {
     return _.includes(VALID_IMAGE_TYPES, ext);
 }
 
-function runLintTask(callback) {
```

Comment:
We should not be running lint on theme directories. Theme developers should install `eslint` run it separately as part of their development process.

Reply:
I agree, the result of a successful lint should have nothing to do with functionality and we don't develop with uploaded theme files so we should have no reason to enforce this on our side.

Reply:
Would be cool if instead of detected `.eslintrc` and running eslint from CLI, if we could detect a generic `lint` npm command and run it. That would allow developers to define their own and keep it entirely separate from CLI, but still preserve this functionality at bundle time. Doesn't have to be done here, but could be a followup task.

Reply:
I agree